src/test/regress/sql/multi_explain.sql

```diff
@@ -6,6 +6,9 @@
 ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 570000;
 ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 570000;
 
+-- print major version to make version-specific tests clear
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+\.\d+') AS major_version;
 
```

Comment:
```SELECT substring(version(), '\d+\.\d+');``` is easier to understand. If you prefer to use server_version, due to less irrelevant data, it'd be better to use ```SELECT substring(current_setting('server_version'), '\d+\.\d+');```. Doing that via \gset seems quite hard to understand.